src/constants/collections.ts

```diff
@@ -0,0 +1,15 @@
+import { SiteKey } from "../types/site"
+
+export const QUESTION_TEMPLATE_COLLECTION_ID = 42
+
+export const SANDBOXED_CUSTOM_ANALYTICS_COLLECTIONS: Record<SiteKey, number> = {
```

Comment:
These defines the 3 isolated collections for each shops.

![CleanShot 2567-09-04 at 02 01 04@2x](https://github.com/user-attachments/assets/0db7b335-042a-441c-8c08-07219e257dfc)